lib/index.js

```diff
@@ -162,16 +170,28 @@ export default class HyperList {
       }
     })
 
-    // Decorate the container element with styles that will match
+    // If using React, the element object turns into a props object.
+    if (config.isReact) {
```

Comment:
I'll give this a shot this weekend and let you know my findings.

> I don't really like all the isReact-specific conditionals necessary.

I didn't read this, but yeah adding framework-specific code isn't that clean :p